main.c

```diff
@@ -390,8 +390,16 @@ static void RenderScreen(SDL_Window *window, SDL_Renderer *renderer, SDL_Texture
     return;
   }
   bool hq = RenderScreenWithPerf(pixels, pitch, g_ppu_render_flags);
-  if (g_display_perf)
+  if (g_display_perf) {
     RenderNumber(pixels + (pitch*2<<hq), pitch, g_curr_fps, hq);
+  }
+  if (g_config.display_perf_title) {
+    char title[] = "The Legend of Zelda: A Link to the Past | FPS: ";
+    char curr_fps[11];
+    int curr_fps_size = sprintf(curr_fps, "%d", g_curr_fps);

```

Comment:
no need for strncat you can just do sprintf(title, "%s%d", kWindowTitle, g_curr_fps);